package.json

```diff
@@ -17,6 +17,7 @@
         "cross-fetch": "^3.1.5",
         "css-loader": "^6.5.1",
         "eslint": "^8.18.0",
+        "eslint-config-prettier": "9.1.0",
```

Comment:
Used exact pins here per the prettier installation docs. Strikes me as a good way to prevent drift across team members installs, which could start polluting our diffs again.